automates/text_reading/src/main/scala/org/clulab/aske/automates/serializer/AutomatesSerializer.scala

```diff
@@ -169,6 +169,11 @@ object AutomatesJSONSerializer {
         val foundBy = json("foundBy").str
         new ContextAttachment(attType, json("contexts").arr, foundBy)
       }
+      case "FunctionAtt" => {
+        val foundBy = json("foundBy").str
+        val trigger = json("trigger").str
+        new ContextAttachment(attType, trigger, foundBy)
```

Comment:
@alicekwak should this be function attachment?

Reply:
I like this idea---gives an indication of what type of function something is. We may still want to add an operator field later to make sure we have clear categories of functions and be able to sort based on those, but this is already quite informative!

Reply:
Yes, this is my first attempt to create a function attachment. I think we can add more to it later as we proceed! (i.e. categories of functions)

Reply:
I think in line 175, you might want to have new FunctionAttachment instead of ContextAttacgment?

Reply:
Yes you're right. Thank you!